### ISSUE TITLE
Some questions about Backbone training

### ISSUE THREAD

**Issue by ruler9527** [valeoai/3DGenZ#2]
Thank you for your work！Could you provide the code of backbone training under your ZSL setting? The code here is only open for testing with trained downloaded backbone models (such as FKAConv).

**Comment by BjoernMichele** [comment]
Thank you for your interest in our work. 

The idea for the training of the backbone is, that we don't change the training process of the original backbones, but instead just change the dataloader, so that point cloud crops are skipped and not shown in the training. So you can use the "normal" training files/ process of the backbones, and use them with our adapted dataloaders for the datasets, to skip the crops of points including unseen point clouds. 

Best, 
Bjoern

**Comment by ZhengLeon** [comment]
Hi, @BjoernMichele，
Thank you for your great work. 
I am trying to reproduce the Full supervision result in Table 2. However, I get some trouble with FKAConv in ScanNet dataset.
Could you please provide some hyperparameters like training epochs num and learning rates for the FSL setting?

Best,
Leon

**Comment by BjoernMichele** [comment]
Dear @ZhengLeon 

Thank you for your interest in our research. 

For the fully supervised setting I recall that we used a starting learning rate of 0.001, with 100 epochs of training. Are you achieving higher values, or lower ? As far as I can recall, for the ZSL setting it was often valuable to have much more epochs of training. 

With best regards, 
Bjoern

**Comment by ZhengLeon** [comment]
Hi @BjoernMichele,
Thanks for your kind reply.
Did you use `freeze_bn() `for the ZSL backbone? I found that if I make the `model.eval` for the backbone, the results would be very low for ZSL backbone. 

Best,
Leon

**Comment by BjoernMichele** [comment]
Dear @ZhengLeon, 

do you mean, if I used freeze_bn() for the backbone pre-training ? There, I did not use it as it would freeze the BN layers in their initialization value. 

Feel also free to write me an email, as normally I catch them earlier than my github notifications. 

Best, 
Björn
